lua/entities/gmod_wire_expression2/core/player.lua

```diff
@@ -85,7 +75,25 @@ e2function string entity:steamID64()
 	if not IsValid(this) then return self:throw("Invalid entity!", "") end
 	if not this:IsPlayer() then return self:throw("Expected a Player but got an Entity!", "") end
 
-	return this:SteamID64() or ""
+	return this:SteamID64()
```

Comment:
I vaguely remember this returning nil if the the player is a bot. Might need the `or ""`. Could be wrong though

Reply:
The rest of the changes look fine

Reply:
Latest update said it now always returns something valid.
> Made Player:SteamID64 and Player:AccountID never return no value

Reply:
`no value` is not the same as `nil`, although [it looks like they did also change the nil case](https://wiki.facepunch.com/gmod/Player:SteamID64~diff:551389). I think the only case where this happened was when a player signed in on a -multirun client.